sys/auto_init/netif/auto_init_ng_at86rf2xx.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/*
+ * @ingroup auto_init_ng_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for nx_at86rf2xx network interfaces
+ *
+ * @author  Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#include "board.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+
+#include "ng_at86rf2xx.h"
+#include "ng_at86rf2xx_params.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#ifdef MODULE_NG_AT86RF2XX
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
+ * @{
+ */
+#define AT86RF2XX_MAC_STACKSIZE     (KERNEL_CONF_STACKSIZE_DEFAULT)
```

Comment:
Actually, the ideal solution would be to compute this number by something like BASE_STACK + DRIVERS_STACK_NEEDS + MAC_LAYER_STACK_NEEDS, where the driver part is defined by the driver and the MAC part is defined by... you get the picture. The question is just if it is possible to define these numbers in a sensible way?